packages/p2p/src/components/app.jsx

```diff
@@ -78,15 +76,6 @@ class App extends React.Component {
                     },
                     [this.setIsAdvertiser, this.setChatInfoUsingAdvertiserInfo]
                 ),
-                order_list_subscription: subscribeWS(
```

Comment:
We should still subscribe to P2P order list from when the P2P component is mounted so that we can observe changes in order statuses (and then `handleNotifications`), however we should **not** do anything with that initial response of that subscribing call (initial response will be the order list).

Reply:
I've re-added the `p2p_order_list` initial subscription and `setP2pOrderList` that will `handleNotifications` with the response list and set `orders` to the same.